spring-cloud-gcp-docs/src/main/asciidoc/spring-integration-pubsub.adoc

```diff
@@ -83,13 +83,29 @@ It requires the channel we just created and a `SubscriberFactory`, which creates
 objects from the Google Cloud Java Client for Pub/Sub.
 The Spring Boot starter for GCP Pub/Sub provides a configured `SubscriberFactory`.
 
-It is also possible to set the message acknowledgement mode on the adapter, which is automatic by
-default.
-On automatic acking, a message is acked with GCP Pub/Sub if the adapter sent it to the channel and
-no exceptions were thrown.
-If a `RuntimeException` is thrown while the message is processed, then the message is nacked.
-On manual acking, the adapter attaches an `AckReplyConsumer` object to the `Message` headers, which
-users can extract using the `GcpPubSubHeaders.ACKNOWLEDGEMENT` key and use to (n)ack a message.
+The `PubSubInboundChannelAdapter` supports three acknowledgement modes, with
+`AckMode.AUTO` being the default value;
+
+Automatic acking (`AckMode.AUTO`)
+
+A message is acked with GCP Pub/Sub if the adapter sent it to the channel and no
+exceptions were thrown. If a RuntimeException is thrown while the message is processed,
```

Comment:
Please, follow the rule "One sentence per line": https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line